[DOC] Provide an example when no framework is used (#680)
elastic/apm-agent-python: docs/how-the-agent-works.asciidoc

Framework integrations require some limited code changes in your app.
E.g. for Django, you need to add `elasticapm.contrib.django` to `INSTALLED_APPS`.

[float]
[[how-it-works-no-framework]]
==== What if you are not using a framework

> In case you are not using a supported framework, for example a simple Python script, you can still

Reviewer: ```suggestion
If you're not using a supported framework, for example, a simple Python script, you can still
```

> leverage the <<automatic-instrumentation,Automatic Instrumentation>>.

Reviewer: ```suggestion
leverage the agent's <<automatic-instrumentation,automatic instrumentation>>.
```

> In order to collect the Spans generated by the supported libraries, you will need to create a Transaction.

Reviewer 2: Can we add a note here about calling `elasticapm.instrument()` in addition to creating a transaction? That's a key requirement for actually getting spans from instrumented libraries.

Author: Hello @basepi ! I've added the requirement to invoke `elasticapm.instrument()` just once.
I am not 100% sure of the wording used.

> It is up to you to define what do you consider a Transaction within your application: it can be the whole execution of the

Reviewer: ```suggestion
It is up to you to determine what you consider a Transaction within your application -- it can be the whole execution of the
```

script or a part of it.

> The example below will consider the whole execution as a single transaction and you'll see 2 HTTP request Spans in it.

Reviewer: ```suggestion
The example below will consider the whole execution as a single transaction with two HTTP request Spans in it.
```
